core/src/main/java/org/radargun/utils/Fuzzy.java

```diff
@@ -12,24 +15,25 @@
 
 /**
  * @author Radim Vansa &lt;rvansa@redhat.com&gt;
+ *
  */
 public final class Fuzzy<T extends Serializable> implements Serializable {
    private Serializable[] values;
-   private double[] probability;
+   private BigDecimal[] probabilities;
 
-   private Fuzzy(Serializable[] values, double[] probability) {
+   private Fuzzy(Serializable[] values, BigDecimal[] probabilities) {
       this.values = values;
-      this.probability = probability;
+      this.probabilities = probabilities;
    }
 
-   public static <T extends Serializable> Fuzzy<T> always(T singleValue) {
-      return new Fuzzy<T>(new Serializable[] {singleValue}, new double[] {1.0});
+   public static <T extends Serializable> Fuzzy<T> uniformEntrySize(T entrySize) {
```

Comment:
I can only see the usage in CacheTestSetupStage and LoadStage. In both places it is used to define uniform entry size. Can you explain where else this can be used? Right now this class is a bit too "fuzzy" to me:) It's too generic, IMO.